language/resource.language.en_gb/strings.po

```diff
@@ -305,7 +305,7 @@ msgid "Select Action"
 msgstr ""
 
 msgctxt "#32013"
-msgid "Updates"
+msgid "Distro Updates"
```

Comment:
"OS Updates" ? `Distro` seems a bit techie/corny...

Reply:
No objections to that. I'll update the PR.